src/main/java/core/basesyntax/randomizer/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.randomizer;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import core.basesyntax.model.Colors;
+import core.basesyntax.model.Figure;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_VALUE = 50;
+
+    public Figure getRandomFigure() {
+        Random random = new Random();
+        ColorSupplier colorSupplier = new ColorSupplier();
+
+        Circle circle = new Circle(random.nextInt(MAX_VALUE));
+        IsoscelesTrapezoid isoscelesTrapezoid = new IsoscelesTrapezoid(random.nextInt(MAX_VALUE),
+                                            random.nextInt(MAX_VALUE), random.nextInt(MAX_VALUE));
+        Rectangle rectangle = new Rectangle(random.nextInt(MAX_VALUE), random.nextInt(MAX_VALUE));
+        RightTriangle rightTriangle = new RightTriangle(random.nextInt(MAX_VALUE),
+                                                        random.nextInt(MAX_VALUE));
+        Square square = new Square(random.nextInt(MAX_VALUE));
+
+        Figure[] figures = {circle, isoscelesTrapezoid, rectangle, rightTriangle, square};
```

Comment:
That's a valid approach, however I see a drawback here - we are creating too many objects, when we need just one. I would recommend to consider another approach, probably using `switch` construction.
Hint: you can create another enum, which will contain all figure names in it.

Reply:
Is it a good practice to make a static method to get a random value, instead of writing "random.nextInt(MAX_VALUE)" every time?

Reply:
@HovorukhaBohdan generally using static methods is not best practice and for our case it will not bring any benefits. We have one `random` instance and each time calling `nextInt` method for random value, that's totally fine :)